tools/tarantool-gdb.py

```diff
@@ -1423,7 +1439,17 @@ def children(self):
                 child_name = str()
                 child_val = (self.val.address.cast(self.ptr_char) + field.bitpos // 8).cast(field.type.pointer()).dereference()
             elif field.name == 'flags':
-                child_val = '0x{:02x}'.format(int(self.val['flags']))
+                flags_val = self.val['flags']
+                child_val = '0x{:02x}'.format(int(flags_val))
+                bits_set = []
+                for bit in range(flags_val.type.sizeof * 8):
+                    if int(flags_val) & (1 << bit) != 0:
+                        if bit in self.known_tuple_flags:
+                            bits_set.append(self.known_tuple_flags[bit])
+                        else:
+                            bits_set.append('*unknown*')
```

Comment:
```suggestion
                        bits_set.append(enumval_name(gdb.lookup_type('tuple_flag'), bit, '*unknown*'))
```
just for illustration, not tested